x/nft/blockchain/init_test.go

```diff
@@ -0,0 +1,70 @@
+package blockchain
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/store"
+)
+
+func TestInitializer(t *testing.T) {
+	const genesis = `
+	{
+		"nft_tokens": [
```

Comment:
naming: "non-fungible token tokens"

Reply:
No, sorry, I just translates what you wrote into human language. It should probably be `nfts`.

How is the structure in a full https://github.com/iov-one/iov-core/blob/v0.8.1/scripts/bnsd/genesis_app_state.json going to look like? It should ne extensible to other NFT types later

Reply:
okay, `nfts` as sibling to `wallets` is great. But then each row needs a specific parser and you need a type/data structure in each row. It is probably easier to go with

```
...
"wallets": [
],
"nfts": {
  "blockchains": [
    {
      "id": "id-123", 
      "owner": "owner-12300000000000",
      "chain":
      ...
    },
  ],
  "usernames": [
  ]
},
"tokens": [
],
```

Then you only need to care about the content of `nfts.blockchains` inside of the ` x/nft/blockchain/` folder and some central code delegates that structure to you.

Reply:
I can make the path `nfts.blockchains` (and allow later addition of `ntfs.usernames`). Extending genesis parser is not an issue.

Do you think `blockchains` is the right key? This structure does not represent a blockchain but a token that will be created within defined blockchain.
I am not opinionated on the naming, just trying to avoid further confusion.

Reply:
blockchains is fine, as it represents the nft name. same goes for usernames.
nfts sound good to me.